pkg/github/client_test.go

```diff
@@ -0,0 +1,27 @@
+package github
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	baseURL   = "http://example.com/base/"
+	uploadURL = "http://example.com/upload/"
+	testToken = "totallyFake"
+)
+
+func TestNewClientFromBearerToken(t *testing.T) {
+	ghc, err := NewClientFromBearerToken(baseURL, uploadURL, testToken)
+	require.NoError(t, err)
+	require.Equal(t, baseURL, ghc.BaseURL.String())
+	require.Equal(t, baseURL, ghc.BaseURL.String())
```

Comment:
Duplicated assertion here -- should we be checking something else?  (`UploadURL`?)

Reply:
Yup! Nice catch. Will fix.

Reply:
Fixed in https://github.com/brigadecore/brigade-github-app/pull/82/commits/1512905043e1e7384da2dc4c934ed0d4c762bf37